src/codegate/providers/copilot/provider.py

```diff
@@ -22,7 +23,8 @@
 )
 from codegate.providers.copilot.streaming import SSEProcessor
 
-logger = structlog.get_logger("codegate")
+setup_logging()
```

Comment:
Do we need this call to `setup_logging()`, I think it might reconfigure the logger  on-the-fly?

Reply:
if we do not set it, copilot loses the scope and is not logging with the expected format